daemon/network.go

```diff
@@ -283,7 +288,40 @@ func (daemon *Daemon) CreateNetwork(create types.NetworkCreateRequest) (*types.N
 	return resp, err
 }
 
+func (daemon *Daemon) createLoadBalancerSandbox(prefix, id string, ip net.IP, n libnetwork.Network, options ...libnetwork.SandboxOption) error {
+
+	c := daemon.netController
+	sandboxName := prefix + "-sbox"
+	sb, err := c.NewSandbox(sandboxName, options...)
+	if err != nil {
+		if _, ok := err.(networktypes.ForbiddenError); !ok {
+			logrus.Errorf("Failed creating %s sandbox: %v", sandboxName, err)
+		}
+		return nil
+	}
+
+	endpointName := prefix + "-endpoint"
+	ep, err := n.CreateEndpoint(endpointName, libnetwork.CreateOptionIpam(ip, nil, nil, nil), libnetwork.CreateOptionLoadBalancer())
+	if err != nil {
+		logrus.Errorf("Failed creating %s in sandbox %s: %v", endpointName, sandboxName, err)
+		return err
+	}
+
+	if err := ep.Join(sb, nil); err != nil {
+		logrus.Errorf("Failed joining %s to sanbox %s: %v", endpointName, sandboxName, err)
+		return err
```

Comment:
These are still going to cause duplicate errors, it should use `errors.Wrapf()` to provide the extra messages with the error.

Reply:
Also: typo s/sanbox/sandbox/

Reply:
Fixed.